monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -1,12 +1,30 @@
-from typing import Optional
+from typing import Optional, List, Dict
 from monitoring.monitorlib.typing import ImplicitDict
 from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest
-import datetime
+from monitoring.monitorlib import fetch
+from monitoring.uss_qualifier.rid import Severity
```

Comment:
We should not import from rid.  If there are common data structures, those should be moved to a common location, and this PR would be an appropriate place to perform that move.

Reply:
Thank you, I have now removed the Severity to a new file and referenced it in both the files.